### PR TITLE
Add password Spraying tool (SMB, OWA, Lync, Cisco ASA etc...)

### PR DESCRIPTION
**Pull request by Fisjkars**
Adapt changes from https://github.com/ThePorgs/Exegol-images/pull/130/files

### CODE REVIEW

File: sources/install/package_ad.sh [hunk 1]
```diff
@@ -718,6 +719,16 @@ function install_pcredz() {
     add-to-list "pcredz,https://github.com/lgandx/PCredz,PowerShell credential dumper"
 }
 
+function install_spray() {
+  # Password Spraying tool for Active Directory Credentials.
```
Comment (QU35T-code) [inline sources/install/package_ad.sh:723]: This comment is not necessary.

File: sources/install/package_ad.sh [hunk 2]
```diff
@@ -718,6 +719,16 @@ function install_pcredz() {
...
+  colorecho "Installing Spray"
+  git -C /opt/tools/ clone https://github.com/Greenwolf/Spray.git
```
Comment (QU35T-code) [inline sources/install/package_ad.sh:725]: Can you add `--depth=1` after the `clone` so that it doesn't download everything ?

**Review by QU35T-code — CHANGES_REQUESTED** [review]
Hey @Fisjkars,

Thanks ! I request some changes.

**Comment by Fisjkars** [comment]
Done

**Review by QU35T-code — APPROVED** [review]

**Review by ShutdownRepo — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
Can you sync with `dev`? It seems you don't have the latest changes made on the workflows, which makes the pipeline fail on the ARM64 runner

**Pull request closed without merging**